apps/frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -30,6 +36,10 @@ const ProtectedRoute: React.FC<Properties> = ({
 		return <Navigate replace to={redirectTo} />;
 	}
 
+	if (!hasAnsweredQuizQuestions && pathname !== AppRoute.QUIZ) {
```

Comment:
Add a case when the onboarding questions are not answered

Reply:
Handling onboarding is already done on the quiz part since i moved it into quiz page for better transition since we only want to show onboarding once.

Reply:
Let me know if I should revert this idea